src/algorithms/pipeline.py

```diff
@@ -0,0 +1,12 @@
+from transformers import pipeline
```

Comment:
Directly using this might not be a great idea because it does not give relevant information as to which algorithm is being used. However, you can refer to [this](https://huggingface.co/transformers/main_classes/pipelines.html#transformers.SummarizationPipeline) and provide detailed information in the UI as to which model is being used.